worker-build/src/main.rs

```diff
@@ -118,19 +118,23 @@ fn use_glue_import() -> Result<()> {
 
 // Bundles the snippets and worker-related code into a single file.
 fn bundle(esbuild_path: &Path) -> Result<()> {
+    let no_minify = env::var("NO_MINIFY").is_ok();
```

Comment:
We should probalbly check for the [NotPresent](https://doc.rust-lang.org/stable/std/env/enum.VarError.html) error variant.

Reply:
Good catch. I made the changes.